### ISSUE TITLE
No official build of v2.5.7-1 for download

### ISSUE THREAD

**Issue by hankliu43** [NVIDIA/nccl#323]
Hi NCCL team,

Looks like there is no official build of v2.5.7-1 for download, at https://developer.nvidia.com/nccl/nccl-download. Do you plan to add it?

**Comment by sjeaugey** [comment]
No, we don't intend to provide an official build for 2.5.7. We tagged it because Tensorflow wanted to use it (see #282), to help keep track of what users are running.

All official builds are reflected to github and tagged so that users can access the corresponding source code, but the opposite is not necessary true. Official builds go through a long and costly qualification process, so we might not want to do it for every small fix, especially when a new version is coming (in this case 2.6.4).

**Comment by stephenmsachs** [comment]
### Issue Cleanup: Helping Us Focus on Current Challenges

  We're reviewing older issues to ensure we prioritize the most relevant and active ones. Since this issue hasn't seen updates in over 6 months, we'll be closing it for now.

  ,*This change helps us focus our efforts on addressing any current issues our users are facing.* If this issue still affects you, please don't hesitate to reopen it with a quick update (e.g., "Still relevant on [version=X]").
  Thanks for your understanding and for contributing to NCCL.
